llama-index-core/llama_index/core/multi_modal_llms/generic_utils.py

```diff
@@ -57,13 +73,14 @@ def image_documents_to_base64(
     for image_document in image_documents:
         if image_document.image:  # This field is already base64-encoded
             image_encodings.append(image_document.image)
-        elif (
+        elif image_document.image_path and is_valid_image_path(
```

Comment:
I don't think we need a dedicated function, this is actually shorter
```suggestion
        elif image_document.image_path and os.path.isfile(
```

Reply:
arguably this check could be done in the `ImageNode` class too 👀

Reply:
> arguably this check could be done in the ImageNode class too 👀

could open another PR do this